lib/snowbridge.ts

```diff
@@ -373,3 +386,37 @@ export function getErrorMessage(err: any) {
   console.error(message, err);
   return message;
 }
+
+export async function assetMetadata(
+  context: Context,
+  env: SnowbridgeEnvironment,
+): Promise<BridgedAssetsMetadata> {
+  const tokens = [
+    ...new Set(
+      env.locations
+        .flatMap((l) => l.erc20tokensReceivable)
+        .map((l) => l.address.toLowerCase()),
+    ),
+  ];
+  const [relaychainNativeAsset, assetMetadataList] = await Promise.all([
+    assets.parachainNativeAsset(context.polkadot.api.relaychain),
+    Promise.all(
+      tokens.map((t) =>
+        assets
+          .assetErc20Metadata(context, t)
+          .then((m) => ({ token: t, metadata: m }))
+          .catch((_) => null),
```

Comment:
```suggestion
          .catch((error) => { 
          console.log(`Error: Asset Metadata ${error}`)
          return null
          }),
```

Reply:
Why not have some error handling here?

Reply:
Nice catch, addressed in https://github.com/Snowfork/snowbridge-app/pull/26/commits/44c6126c865b20ba04771c89f319d1ae37afae09